src/main.ts

```diff
@@ -27,8 +27,11 @@ export default class TTSPlugin extends Plugin {
 		this.addCommand({
 			id: 'start-tts-playback',
 			name: 'Start playback',
-			editorCallback: (_, view) => {
-				this.ttsService.play(view);
+			checkCallback: (checking: boolean) => {
```

Comment:
command used to be available only in editor mode. I changed the callback so it can be found in the view mode as well.